module/plugins/internal/misc.py

```diff
@@ -592,6 +592,12 @@ def parse_time(value):
     if re.search("da(il)?y|today", value):
         seconds = seconds_to_midnight()
 
+    elif re.search("\d:\d\d", value):
+        # use the HH:MM:SS format, NOTE: when only one ':' is found, it assumes MM:SS
```

Comment:
Well, there is no way around assuming one of the two without context, and seeing that I stumbled upon that while handling wait times (they were always below 1h), this seemed the obvious choice. However, to be more general, maybe an argument should be added to the function telling it how to interprete a single colon.